content/tutorials/single-page-tutorials/wallet-setup.md

```diff
@@ -0,0 +1,431 @@
+# Setup testnet address and receive funds from faucet
+
+In this tutorial you will create a new mnemonic seed phrase in order to setup a Stronghold account with the [NodeJS binding of wallet.rs](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/nodejs). Afterwards you will generate a new address, request funds from the Shimmer testnet faucet and check your account balance.
+
+## Prerequisites
+
+- Development Environment and Libraries
+  - Installed [Node.js](https://nodejs.org/en/)
+
+***
+
+## Setup
+
+### Prepare development environment
+
+Create a new folder for the tutorial in a location of your choice and navigate to it:
+
+```bash
+mkdir wallet-setup
+cd wallet-setup
+```
+
+Run the Node.js initializer to configure the project:
+
+```bash
+npm init --yes
+```
+
+Overwrite the `package.json` file with the following content:
+
+```yaml
+{
+  "name": "wallet-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "create-mnemonic.js",
+  "dependencies": {
+    "@iota/wallet": "2.0.2-alpha.21",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1",
+    "node-fetch": "^2.6.7"
+  },
+
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+
+Install new dependencies:
+:::note
+
+Installation might take a while, if there is no prebuilt wallet NPM package for your combination of OS and NodeJS version.
+
+:::
+```bash
+npm install
+```
+
+***
+
+### Prepare network configuration
+
+Create a new file `networkConfig.js` and add the following code:
+
+```javascript
+var networkConfig = {};
+
+// Shimmer Beta network configuration
+networkConfig.node = 'https://api.testnet.shimmer.network';
+networkConfig.faucetWebsite = 'https://faucet.testnet.shimmer.network';
+networkConfig.faucetApi = 'https://faucet.testnet.shimmer.network/api/enqueue';
+networkConfig.explorer = 'https://explorer.shimmer.network/testnet';
+
+module.exports = { networkConfig };
+```
+
+***
+
+### Prepare environment variables
+
+Create a new file `.env` and add the content below:
+
+```javascript
+ACCOUNT_NAME = "<Enter_your_name_here>"
+SH_PASSWORD = "<Enter_your_password_here>"
+MNEMONIC = "<You_will_create_your_own_mnemonic_seed_phrase_in_the_next_step_and_paste_it_in_here>"
+```
+
+Enter your desired account name as well as a secure Stronghold password. Your new mnemonic seed phrase will be created in the next step and you will paste it in here afterwards.

```

Comment:
```suggestion
Enter your desired account name as well as a secure Stronghold password. You will create your new mnemonic seed phrase in the next step and paste it here afterward.
```
I get that you want to separate the setup phase and scripts, but it may be easier for the users if they generate the mnemonic before the .env so they don't have to go back